README.md

```diff
@@ -60,5 +60,64 @@ To register a device, run as root from LmP device:
 (i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+In scenarios where you'd like to test the API call without involving a physical device or emulation:
+
+This project includes a simple [fake-lmp-device-register](./fake-lmp-device-register) tool. 
+This tool mimics the behavior of `lmp-device-register`, enabling you to quickly test this project's API 
+without a real device.
+
+## Running the fake-lmp-device-register
+
+### Dependencies:
+- Python 3
+- `requests` library
+- `openssl` command-line tool
+- `curl`
+
+### Usage:
+
+1. Install the required Python library using pip:
+
+```shell
+pip3 install requests
+```
+
+2. Execute the fake-lmp-device-register tool:
+```shell
+python3 fake-lmp-device-register --registration-url "http://<IP of docker-compose host>:80/sign" --factory <factory>
+```
+
+3. Verify Connectivity with the Server:
+
+After successfully executing the fake-lmp-device-register, a new directory, `var/sota`, will be created in the current 
+directory. This contains a curl script and the necessary certificates.
+
+Navigate to this directory:
+
+```shell
+cd var/sota
+```
+
+Inspect the server URL configured in the `sota.toml` file:
+
+```shell
+$ cat sota.toml 
+```
+
+You should see an output similar to:
+
+```shell
+[tls]
+server = "https://ec056c71-d698-41a6-bea0-1f1ac8c44bde.ota-lite.foundries.io:8443"
+ca_source = "file"
+pkey_source = "file"
+cert_source = "file"
+...
+```
+
+Now, test the connectivity with the server using curl:
+
+```shell
+curl -v --cacert root.crt --cert client.pem --key pkey.pem https://ec056c71-d698-41a6-bea0-1f1ac8c44bde.ota-lite.foudries.io:8443/
```

Comment:
@vkhoroz please let me know if that was what you suggested?
Have you a better idea over how to do this check or would you like to propose any specific anypoint?

Reply:
It may work as one of the options.

I was specifically referring to [this file](https://github.com/foundriesio/factory-registration-ref/blob/main/fake-lmp-device-register#L100) created by the `fake-lmp-device-register`, which aliases all the necessary parameters, so that one can run smth like e.g. `${SOTA_DIR}/curl https://${REPO_ID}.ota-lite.foundries.io:8443/repo/root.json`.